examples/migrate-pvc/README.MD

```diff
@@ -0,0 +1,125 @@
+# Migrate a PVC from a kubernetes cluster to another one 
+
```

Comment:
I was thinking should the title be, Migrating volume data  or PVC data from a cluster to another.

Reply:
if we change this, we will also have to change the dir name where this README is at.

Reply:
You want to add "volume data" because you think there could be some confusion about just migrating the spec of the pvc without the data that come along ? If that's the case I would prefer saying "Migrating the volume data **of** a PVC from a cluster to another". This is what we're doing actually, we are not migrating the pvc spec, we recreate manually an empty pvc on the destination cluster. Does it work for you ?